docker/puppetserver-base/docker-entrypoint.d/90-log-config.sh

```diff
@@ -0,0 +1,20 @@
+#! /bin/sh
+
+### Print configuration for troubleshooting
+echo "System configuration values:"
+# shellcheck disable=SC2039 # Docker injects $HOSTNAME
+echo "* HOSTNAME: '${HOSTNAME}'"
+echo "* hostname -f: '$(hostname -f)'"
+echo "* PUPPETSERVER_HOSTNAME:PUPPET_MASTERPORT: '${PUPPETSERVER_HOSTNAME}:${PUPPET_MASTERPORT}'"
+echo "* DNS_ALT_NAMES: '${DNS_ALT_NAMES}'"
+echo "* SSLDIR: '${SSLDIR}'"
```

Comment:
Not quite there yet...adding in the SSLDIR env var to the Dockerfile. And the LOGDIR env var while I'm at it.

Reply:
Took more than just that... also had to call `puppetserver ca setup` to make sure certs were put down on disk before logging info on them.